velox/experimental/cudf-exchange/CudfExchangeSource.cpp

```diff
@@ -65,6 +65,11 @@ std::shared_ptr<CudfExchangeSource> CudfExchangeSource::create(
 
 void CudfExchangeSource::process() {
 
+  if (closed_.load() == true) {
```

Comment:
Load is implicitly called when accessing an std::atomic, so this might be a bit more readable
```suggestion
  if (closed_) {
```